network/NETWORK-01-MULTIROOT.DATA

```diff
@@ -0,0 +1,379 @@
+SCHEDULE
+------------------------------------------------------------------------------------------------
+
+RPTRST
+ 'BASIC=2' /
+
+
+INCLUDE
+ 'include/MODEL02-D-1.inc' /
+
+INCLUDE
+ 'include/flowline_netbalance.inc' /
+
+TUNING
+ 0.1 2 /
+ /
+ 25 1 50 1 30 30 /
+
+-- start date:  01  'NOV' 2018 /
+
+GRUPTREE
+   'INJE'     'FIELD'  /
+   'GRPA'  'FIELD' /
+   'GRPB'  'GRPA' /
+   'PRODA'  'GRPA'  /
+   'PRODB'  'GRPB'  /
+/
+
+WELSPECS
+ 'PROD1'  'PRODA'   6    3   1*      'OIL'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+ 'PROD2'  'PRODB'  10   4	1*	'OIL'  0.00	 'STD'     'SHUT'      'YES'	0      'SEG' /
+ 'PROD3'  'PRODA'  6 19   1*      'OIL'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+ 'INJ1'   'INJE'   2    13   1*    'WATER'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+ 'INJ2'  'INJE'     12 20   1*    'WATER'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+/
+
+COMPDAT
+ 'PROD1'   6   3   7   7      'OPEN'   0      6.145  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD1'   6   3   8   8      'OPEN'   0      9.703  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD1'   6   3   9   9      'OPEN'   0    184.170  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD1'   6   3   10  10     'OPEN'   0     24.709  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD2'   10   4   7   7      'OPEN'	0    20.473  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD2'   10   4   8   8      'OPEN'	0    17.619  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD2'   10   4   9   9      'OPEN'	0    80.721  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD2'   10   4   10  10     'OPEN'	0    11.755  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'PROD3'   6 19   7   7      'OPEN'   0     21.629  0.216  1*	0.00	0.0 'Z' 1* /
+ 'PROD3'   6 19   8   8      'OPEN'   0     12.441  0.216  1*	0.00	0.0 'Z' 1* /
+ 'PROD3'   6 19   9   9      'OPEN'   0    178.588  0.216  1*	0.00	0.0 'Z' 1* /
+ 'PROD3'   6 19   10  10     'OPEN'   0     11.149  0.216  1*	0.00	0.0 'Z' 1* /
+ 'INJ1'    2   13   1  1      'OPEN'   0    138.457  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'INJ1'    2   13   2  2      'OPEN'   0      0.760  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'INJ1'    2   13   3  3      'OPEN'   0     53.129  0.216  1*   0.00	 0.0 'Z' 1* /
+ 'INJ1'   2   13    8   8     'OPEN'   0    20.547  0.216  1*	0.00	0.0 'Z' 1* /
+ 'INJ1'   2   13    9   9     'OPEN'   0    45.709  0.216  1*	0.00	0.0 'Z' 1* /
+ 'INJ1'   2   13   10  10     'OPEN'   0    25.267  0.216  1*	0.00	0.0 'Z' 1* /
+ 'INJ1'   2   13   11  11     'OPEN'   0    33.982  0.216  1*	0.00	0.0 'Z' 1* /
+ 'INJ2'   12 20   9   9     'OPEN'   0    139.042  0.216  1*   0.00    0.0 'Z' 1* /
+ 'INJ2'   12 20  10  10     'OPEN'   0     18.620  0.216  1*   0.00    0.0 'Z' 1* /
+ 'INJ2'   12 20  11  11     'OPEN'   0     77.048  0.216  1*   0.00    0.0 'Z' 1* /
+/
+
+BRANPROP
+  'PRODA'  'GRPA'    9999  /
+  'PRODB'  'GRPB'    2    /
+  'GRPB'   'GRPA'    9999 /
+  'GRPA'   'FIELD'   2    /
+	'GRPA'   'FIELD'   9999 /
```

Comment:
Not totally sure whether you want the two cases to have identical network setup.  If yes, `	'GRPA'   'FIELD'   9999 /` might be an accident.